src/client.rs

```diff
@@ -74,6 +76,164 @@ pub extern "C" fn rustls_client_config_builder_build(
     }
 }
 
+/// Currently just a placeholder with no accessors yet.
+/// https://docs.rs/rustls/0.19.0/rustls/struct.RootCertStore.html
+#[allow(non_camel_case_types)]
+pub struct rustls_root_cert_store {
+    _private: [u8; 0],
+}
+
+/// Input to a custom certificate verifier callback. See
+/// rustls_client_config_builder_dangerous_set_certificate_verifier().
+#[allow(non_camel_case_types)]
+#[repr(C)]
+pub struct rustls_verify_server_cert_params<'a> {
+    end_entity_cert_der: rustls_slice_bytes<'a>,
+    intermediate_certs_der: &'a rustls_slice_slice_bytes<'a>,
+    roots: *const rustls_root_cert_store,
+    dns_name: rustls_str<'a>,
+    ocsp_response: rustls_slice_bytes<'a>,
```

Comment:
nit: It's interesting that these struct fields are public from the point of view of a C client, but a Rust client using `crustls` (for some reason) wouldn't be able to access them. Should they all be `pub` if only for clarity of intent? Does that confuse `cbindgen` in any way?